### ISSUE TITLE
ModuleNotFoundError: No module named 'future'

### ISSUE THREAD

**Issue by pendolf** [namespace-ee/django-datadog-logger#2]
* Django DataDog Logger version: "0.3.3"
* Python version: "3.6"
* Operating System: docker Debian 10

### Description

After configuring logging and trying to start django server I'v got the error below. Seems like future module is missing in requirements.

### What I Did

```
core-api_1            | Traceback (most recent call last):
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 390, in resolve
core-api_1            |     found = getattr(found, frag)
core-api_1            | AttributeError: module 'django_datadog_logger.formatters' has no attribute 'datadog'
core-api_1            | 
core-api_1            | During handling of the above exception, another exception occurred:
core-api_1            | 
core-api_1            | Traceback (most recent call last):
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 392, in resolve
core-api_1            |     self.importer(used)
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django_datadog_logger/formatters/datadog.py", line 14, in <module>
core-api_1            |     import django_datadog_logger.celery
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django_datadog_logger/celery.py", line 7, in <module>
core-api_1            |     from django_datadog_logger.local import Local  # NOQA
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django_datadog_logger/local.py", line 11, in <module>
core-api_1            |     from future import standard_library
core-api_1            | ModuleNotFoundError: No module named 'future'
core-api_1            | 
core-api_1            | The above exception was the direct cause of the following exception:
core-api_1            | 
core-api_1            | Traceback (most recent call last):
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 545, in configure
core-api_1            |     formatters[name])
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 656, in configure_formatter
core-api_1            |     result = self.configure_custom(config)
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 471, in configure_custom
core-api_1            |     c = self.resolve(c)
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 399, in resolve
core-api_1            |     raise v
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 392, in resolve
core-api_1            |     self.importer(used)
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django_datadog_logger/formatters/datadog.py", line 14, in <module>
core-api_1            |     import django_datadog_logger.celery
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django_datadog_logger/celery.py", line 7, in <module>
core-api_1            |     from django_datadog_logger.local import Local  # NOQA
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django_datadog_logger/local.py", line 11, in <module>
core-api_1            |     from future import standard_library
core-api_1            | ValueError: Cannot resolve 'django_datadog_logger.formatters.datadog.DataDogJSONFormatter': No module named 'future'
core-api_1            | 
core-api_1            | During handling of the above exception, another exception occurred:
core-api_1            | 
core-api_1            | Traceback (most recent call last):
core-api_1            |   File "busgo/manage.py", line 15, in <module>
core-api_1            |     execute_from_command_line(sys.argv)
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
core-api_1            |     utility.execute()
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 357, in execute
core-api_1            |     django.setup()
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django/__init__.py", line 19, in setup
core-api_1            |     configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
core-api_1            |   File "/usr/local/lib/python3.6/site-packages/django/utils/log.py", line 76, in configure_logging
core-api_1            |     logging_config_func(logging_settings)
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 802, in dictConfig
core-api_1            |     dictConfigClass(config).configure()
core-api_1            |   File "/usr/local/lib/python3.6/logging/config.py", line 548, in configure
core-api_1            |     'formatter %r: %s' % (name, e))
core-api_1            | ValueError: Unable to configure formatter 'json': Cannot resolve 'django_datadog_logger.formatters.datadog.DataDogJSONFormatter': No module named 'future'
web-admin_core-api_1 exited with code 1

Process finished with exit code 0
```

**Comment by lnagel** [comment]
Fixed by https://github.com/namespace-ee/django-datadog-logger/commit/a22dbc4cd2b0075d71211a291182a83cb4e1cd4b

**Comment by lnagel** [comment]
Released as 0.3.4
